jans-cedarling/bindings/cedarling_python/example.py

```
@@ -331,10 +332,71 @@ def authorize_without_token():
+        "app_id": "application_id",
+        "name": "Some Application",
+        "url": {
+            "host": "jans.test",
+            "path": "/protected-endpoint",
+            "protocol": "http"
+        }
+    })
+
+    # Build typed tokens coming potentially from different issuers
+    # Use locally defined sample payloads so this example is self-contained
+    token_a = "eyJhbGciOi...access_token_sample"
```

Ruff (0.14.2)

Possible hardcoded password assigned to: "token_a"

(S105)